### ISSUE TITLE
Runtime Error Triggered on Ribbon Reload During Debugging Process

### ISSUE THREAD

**Issue by gnyuan** [cunnane/xloil#103]
I'm currently facing an issue with debugging the ribbon in my project. I've successfully connected Visual Studio Code to the debugpy port, but when I make changes to a function(does not change xml code) and save it, xloil reports a Runtime Error.

```
[11:47:57] [error] During Event D:\workspace\xloil\Tools.py: RuntimeError: RuntimeError: Internal: could not find addin associated with current thread
Traceback (most recent call last):
  File "D:\workspace\xloil\.venv\Lib\site-packages\xloil\importer.py", line 97, in _on_file_modified
    self.reload(module)
  File "D:\workspace\xloil\.venv\Lib\site-packages\xloil\importer.py", line 70, in reload
    return self._reloader(module, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\workspace\xloil\.venv\Lib\site-packages\xloil\_superreload.py", line 237, in superreload
    module = reload(module)
             ^^^^^^^^^^^^^^
  File "D:\Program Files\Python312\Lib\importlib\__init__.py", line 131, in reload
    _bootstrap._exec(spec, module)
  File "<frozen importlib._bootstrap>", line 866, in _exec
  File "D:\workspace\xloil\.venv\Lib\site-packages\xloil\importer.py", line 360, in exec_module
    super().exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "D:\workspace\xloil\Tools.py", line 112, in <module>
    _ribbon_ui = xlo.ExcelGUI(ribbon=r'''
                 ^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: Internal: could not find addin associated with current thread
```

Thank you for your time and assistance.

**Comment by gnyuan** [comment]
Observed that `xlo.func` updates reload correctly, but `ribbon code` changes consistently trigger a RuntimeError. Need a quick investigation and fix.

**Comment by cunnane** [comment]
This should be fixed in xlOil 0.18.8, just released.

**Comment by gnyuan** [comment]
Thanks, Nice Code!!
